gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/jupiter/reactor/v4/subscription/DefaultSubscriptionDispatcher.java

```diff
@@ -50,21 +63,37 @@ public class DefaultSubscriptionDispatcher extends AbstractService<SubscriptionD
 
     public static final int ON_SUBSCRIPTION_ERROR_RETRY_COUNT = 5;
     public static final int ON_SUBSCRIPTION_ERROR_RETRY_DELAY_MS = 3_000;
+
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultSubscriptionDispatcher.class);
     private static final String SUBSCRIPTION_ENTRYPOINT_FIELD = "entrypointId";
+
+    protected static final String TRACING_SPAN_NAME = "SUBSCRIPTION";
+    protected static final String ATTR_INTERNAL_TRACING_SPAN = "subscription-tracing-span";
```

Comment:
Is this attribute used somewhere else? If so, could you move it to InternalContextAttribute as Constant?

Reply:
No, it's used internally only (and in junit test)